content/docs/user-guide/programming/gems/code-gems.md

```diff
@@ -43,3 +43,23 @@ The `<Gem>EditorModule` class contains the logic for using the Gem in the Editor
 ### `<Gem>EditorSystemComponent.cpp`, `<Gem>EditorSystemComponent.h`
 
 The `<Gem>EditorSystemComponent` class is the system component class for managing the Editor module. You can use this class to program Editor functionality. For example, if your Gem implements a component or a window that appears in the O3DE Editor.
+
+## Building a Gem
```

Comment:
Correct, creating a gem automatically registers it with either the `project.json` if the gem is created under the project directory, `engine.json` if the gem is created under the engine directory, otherwise it registers the gem wotj `o3de_manifest.json`.